java/ql/src/Security/CWE/CWE-312/AllowBackupAttributeEnabled.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Android allowBackup attribute enabled
+ * @description
+ * @kind problem
+ * @problem.severity recommendation
+ * @security-severity 7.5
+ * @id java/android/allowBackup-attribute-enabled
```

Comment:
This needs to be all lowercase. Also, I think we can make it shorter.

```suggestion
 * @id java/android/allowbackup-enabled
```

Ref: https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-id-id